src/kafka/base.rs

```diff
@@ -358,8 +480,8 @@ pub async fn consume_partitions(
     client_config
         .set("bootstrap.servers", &survey_config.consumer)
         .set("security.protocol", "SASL_PLAINTEXT")
-        .set("group.id", group_id)
-        .set("debug", "consumer,cgrp,topic,fetch");
+        .set("group.id", group_id);
+    // .set("debug", "consumer,cgrp,topic,fetch"); // Seems to ignore EnvFilter directives
```

Comment:
Same here. Want to leave commented code in the codebase?

Reply:
Likewise, are you ok with this?
```suggestion
    // To get logs from kafka, add .set("debug", "consumer,cgrp,topic,fetch").
    // Setting an EnvFilter directive, e.g., RUST_LOG=rdkafka=debug, doesn't work.
```